js/wp-seo-term-scraper.js

```diff
@@ -44,11 +44,28 @@
 				if ( elem !== null ) {
 					val = elem.value;
 				}
+				if ( val === '' ) {
+					val = wpseoTermScraperL10n.metadesc_template;
+				}
+				break;
+			case 'snippetMeta':
+				elem = document.getElementById( 'yoast_wpseo_metadesc' );
+				if ( elem !== null ) {
+					val = elem.value;
+				}
 				break;
 			case 'text':
 				val = this.getContentTinyMCE();
 				break;
 			case 'pageTitle':
+				val = document.getElementById( 'hidden_wpseo_title' ).value;
+				if ( val === '' ) {
+					val = wpseoTermScraperL10n.title_template;
+				}
+				if (val === '' ) {
+					val = '%%title%% - %%sitename%%';
```

Comment:
We choose to do this in the JavaScript in stead of PHP to determine the output in the scraper.